pandas/tests/io/parser/dtypes/test_dtypes_basic.py

```diff
@@ -335,3 +336,29 @@ def test_nullable_int_dtype(all_parsers, any_int_ea_dtype):
     )
     actual = parser.read_csv(StringIO(data), dtype=dtype)
     tm.assert_frame_equal(actual, expected)
+
```

Comment:
Might be also good to (smoke) test the error if the default type isn't a valid pandas_type